pkg/process/metadata/parser/service.go

```diff
@@ -38,6 +39,10 @@ var binsWithContext = map[string]serviceExtractorFn{
 	"sudo":      parseCommandContext,
 }
 
+var serviceTagCommands = map[string]struct{}{
+	kubeletPrefix: {},
+}
```

Comment:
[naming] From what I can tell you're trying to do an exact exe match, so let's not call it `serviceTagsCommands` as it's not apparent what it's doing. Maybe `exactExes` and some kind of comment to document what you're aiming to accomplish?
Also `kubeletPrefix` is inaccurate there's no prefix. I would say to just avoid the constant altogether and just do 
```
kubelet : {}
```